### PR TITLE
Feature 1569/loading feedback requests

### PR DESCRIPTION
**Pull request by smithj11**
The View Feedback page, Received Request page, and View Responses component now have skeleton loaders.

### CODE REVIEW

File: web-ui/src/components/skeleton_loader/SkeletonLoader.jsx [hunk 1]
```diff
@@ -14,51 +16,144 @@ const StyledCard = styled(Card)(() => ({
...
+        />
+        <Box mt={2} ml={2} mr={2} height={80}>
+          <Skeleton variant="text" height={15} />
+          <Box height={10} />
+          <Skeleton variant="text" height={15} />
+          <Skeleton variant="text" width={180} height={15} />
+        </Box>
+      </StyledCard>
+    );
+  }
```
Comment (mvolkmann) [inline web-ui/src/components/skeleton_loader/SkeletonLoader.jsx:54]: Is Prettier being used to format code?

Comment (smithj11) [reply web-ui/src/components/skeleton_loader/SkeletonLoader.jsx:54]: I haven't used it, but I can go back and use Prettier on the code I wrote if you wish!

Comment (mvolkmann) [reply web-ui/src/components/skeleton_loader/SkeletonLoader.jsx:54]: Nearly all OCI web apps use Prettier to format JavaScript and TypeScript code, so I think it's a good idea.

Comment (smithj11) [reply web-ui/src/components/skeleton_loader/SkeletonLoader.jsx]: I used Prettier on the skeleton loader file in question, and will use it from now on before pushing anything. Thank you for the advice!

**Review by mkimberlin — APPROVED** [review]

**Pull request merged**